plugin/kubernetes/object/endpoint.go

```diff
@@ -173,6 +173,9 @@ func EndpointSliceV1beta1ToEndpoints(obj meta.Object) (meta.Object, error) {
 	}
 
 	for _, end := range ends.Endpoints {
+		if end.Conditions.Ready == nil || !*end.Conditions.Ready {
+			continue
+		}
```

Comment:
Looks like we're missing a commit to add this clause to `EndpointSliceToEndpoints`.

Reply:
Whoops, I missed a commit! 
Fixed. Thanks!